ci/nightly/pipeline.yml

```diff
@@ -590,6 +591,17 @@ steps:
       - ./ci/plugins/mzcompose:
           composition: cloud-canary
 
+  - id: output-consistency
+    label: "Output consistency"
+    timeout_in_minutes: 58
+    agents:
+      queue: linux-x86_64
+    artifact_paths: junit_*.xml
+    plugins:
+      - ./ci/plugins/mzcompose:
+          composition: output-consistency
+          args: ["--seed=$BUILDKITE_JOB_ID", "--runtime-in-sec=3300", "--dry-run=False", "--fail-fast=False", "--execute-setup=True"]
```

Comment:
For the boolean parameters I think `--no-dry-run`, `--no-fail-fast`, `--execute-setup` seem cleaner. But also, aren't those the defaults?

Reply:
✅ That's true, removed.